css-grid-classes/style.css

```diff
@@ -0,0 +1,36 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
```

Comment:
The container does not need to have display flex, flex wrap, or min height. The height should be dictated by content!

Reply:
ahh i see ok. when i take them out it looks exactly the same. I think i was confused because of the reference sheet. I think i read it incorrectly 
![image](https://user-images.githubusercontent.com/52356630/98996534-dc7a9b00-24e7-11eb-99ea-0006f2465f32.png)

Reply:
it says properties for parent. are they referring to the row div container then? and not the entire container